### PR TITLE
add changelogs

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

 
- Docs Pull Request
 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/CHANGELOG.rst [hunk 1]
```diff
@@ -3,3 +3,12 @@ Ansible Network Collection Release Notes
...
 .. contents:: Topics
+
+
+v1.0.0
+======
+
+Bugfixes
+--------
+
+- test
```
Comment (Qalthos) [inline changelogs/CHANGELOG.rst:14]: Did you mean to leave this here? Maybe a release_summary section would be better?

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: changelogs/CHANGELOG.rst (hunk 1 above)
Comment (rohitthakur2590) [reply changelogs/CHANGELOG.rst]: agreed!! updated the same.

File: meta/runtime.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+requires_ansible: '>=2.9.10,<2.11'
```
Comment (pabelanger) [inline meta/runtime.yml:2]: please uncap this

Comment (rohitthakur2590) [reply meta/runtime.yml]: updated

**Review by Qalthos — APPROVED** [review]

**Pull request merged**